scripts/files/file_tiff.py

```diff
@@ -36,15 +36,36 @@ def __init__(
         self._srs: Optional[bytes] = None
 
     def set_srs(self, srs: bytes) -> None:
+        """Set the srs.
+
+        Args:
+            srs: the srs in bytes
+        """
         self._srs = srs
 
     def set_scale(self, scale: int) -> None:
+        """Set the scale.
```

Comment:
I think this could use further explanation, what is a `scale` what are normal values for it?